code/modules/reagents/dispenser/dispenser2.dm

```diff
@@ -181,7 +181,7 @@
 	else if(href_list["ejectBeaker"])
 		if(container)
 			var/obj/item/chems/B = container
-			B.dropInto(loc)
+			user.put_in_hands(B)
```

Comment:
This is actually problematic. If the user is not near the machine, this will teleport the item to their location and leave it there. Since this is in Topic, there's no control over where the user is. It leads to common borg and AI bugs. You need to check `CanPhysicallyInteract` if you want to do this.

Reply:
Understood